script/bootstrap.py

```diff
@@ -110,6 +110,9 @@ def parse_args():
                       help='The static library path of libchromiumcontent.')
   parser.add_argument('--defines', default='',
                       help='The build variables passed to gyp')
+  parser.add_argument('--sccache',
+                      help='Enables sccache support, provide the path to ' +
+                        'sccache.  E.g.  $(which sccache)')
```

Comment:
[PEP8](http://pep8online.com) complains about the indentation here.
It says that the third line should have the same indentation as the second.
```python
help='Enables sccache support, provide the path to ' +
'sccache. E.g. $(which sccache)')
```

I would also ask to to remove double spaces around `E.g.` =)